lib/Config.js

```diff
@@ -191,10 +191,32 @@ function hdClientLocationConstraintAssert(configHd) {
     return hdclientFields;
 }
 
+function dmfLocationConstraintAssert(location, locationObj) {
```

Comment:
```suggestion
function dmfLocationConstraintAssert(locationObj) {
```
`location` parameter is not needed here!

Reply:
Hi Naren, true, it has been removed, I forgot to answer this comment, please see https://github.com/scality/cloudserver/pull/4532/files#diff-b06d72fe5b9eba1ab27b65f49f4fc461894ee5bf7507753734fdb5fff1eee387R194